Makefile

```diff
@@ -0,0 +1,17 @@
+
+.PHONY: depinstall clean sdist test
+
+clean:
+	rm -rf sdc/rabbit/doc/html
+	rm -v dist/sdc-rabbit-python-*.tar.gz
+
+depinstall:
+	pip3 install -r requirements.txt
+
+sdist:
```

Comment:
Strictly an `sdist` in the python world, but the step may be more commonly understood as `dist`